app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt

```diff
@@ -92,7 +89,24 @@ class GardenFragment : Fragment() {
     }
 
     private fun onSelectionChanged(selectedCount: Int) {
-        // TODO: Enable/disable ActionMode when items are selected.
+        if (selectedCount == 0) {
+            actionMode?.let {
+                it.finish()
+                actionMode = null
+            }
+        } else {
+            (requireActivity() as? AppCompatActivity)?.let { activity ->
+                if (actionMode == null) {
+                    actionMode = activity.startSupportActionMode(GardenActionModeCallback())
+                }
+            }
+            actionMode?.title =
+                resources.getQuantityString(
+                    R.plurals.garden_action_mode_count,
+                    /* quantity= */ selectedCount,
```

Comment:
I think it's ok to omit the param comments (they're not mentioned during other usages of `getQuantityString`)

Reply:
Done.